system/models/shopping/order.py

```diff
@@ -19,4 +19,4 @@ def get_customer(self) -> Customer:
         return self._customer
 
     def get_order_as_list(self) -> List:
-        return [self._order_id, self._order_items, self._customer]
+        return [self._order_id, self._order_items, self._customer]
```

Comment:
I think you committed this by accident? It removed the last newline at the end of the order file. You should delete it from the push.